docs/xcm/building-with-xcm/xc-reserve-transfer.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 7
```

Comment:
Should `xc-assets-withdraw.md` page also be on sidebar then?

Reply:
Definitely